### PR TITLE
throw error if config is specified in pages

### PR DESCRIPTION
**Pull request by Maximo-Guk**
#1067

### CODE REVIEW

File: packages/wrangler/src/pages.tsx [hunk 1]
```diff
@@ -1507,6 +1512,11 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
           return;
         }
 
+        if (config) {
+          logger.error("Pages does not support wrangler.toml");
```
Comment (threepointone) [inline packages/wrangler/src/pages.tsx:1516]: This won't throw an error, you should use `throw new Error("...` instead

**Comment by threepointone** [comment]
Please add a changeset and tests, thank you!

**Comment by threepointone** [comment]
Could you add a changeset and a test please? I'd be happy to review and land it then!

**Review by petebacondarwin — APPROVED** [review]
LGTM!

**Review by GregBrimble — APPROVED** [review]

**Pull request merged**